acir/src/circuit/opcodes/black_box_function_call.rs

```diff
@@ -76,6 +76,16 @@ pub enum BlackBoxFuncCall {
         inputs: Vec<FunctionInput>,
         outputs: Vec<Witness>,
     },
+    VerifyProof {
+        key: Vec<FunctionInput>,
+        proof: Vec<FunctionInput>,
+        public_inputs: Vec<FunctionInput>,
```

Comment:
Add a comment here specifying what these `public_inputs` refer to

Reply:
Comments added